weather_dl_v2/cli/app/cli_config.py

```diff
@@ -50,6 +50,8 @@ def from_dict(cls, config: t.Dict):
 
 def get_config():
     global cli_config
+    # TODO: Update this so cli can work from any folder level.
+    # Right now it only works in folder where cli_config.json is present
```

Comment:
nit: add `.`.

Reply:
updated in [ad2d386](https://github.com/google/weather-tools/pull/407/commits/ad2d3869472a5410ba6dda75dec46eb217cbf6f7)